tests/core/Jalasoft.TeamUp.Projects.Core.Tests/Validators/ProjectValidatorTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿namespace Jalasoft.TeamUp.Projects.Core.Tests.Validators
+{
+    using System;
+    using FluentValidation.TestHelper;
+    using Jalasoft.TeamUp.Projects.Core.Validators;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Xunit;
+
+    public class ProjectValidatorTests
+    {
+        private readonly ProjectValidator validator;
+
+        public ProjectValidatorTests()
+        {
+            this.validator = new ProjectValidator();
+        }
+
+        [Fact]
```

Comment:
please use the naming convention the team has been using so far: MethodName_Context_ExpectedResult

Reply:
The conventions were renamed